src/main/java/com/controller/QuoteController.java

```diff
@@ -73,4 +73,34 @@ public ResponseEntity deleteQuote(@PathVariable int id) {
 
     }
 
+    /**
+     * Adds one like to the quote with the specified id
+     *
+     * @param id
+     * @return
+     */
+    @GetMapping(path = "/quotes/{id}/like")
+    public ResponseEntity likeQuote(@PathVariable int id){
+        //add 1 like to quote rating
+        //get the quote current rating
+        quoteService.addRating(id);
+
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    /**
+     * Removes one like to the quote with the specified id
```

Comment:
how it's done is implementation detail, maybe better to just say "Add dislike to specified quote"

Reply:
Fixed in neutralizer/fixes-for-review-for-model-expanding-branch